calls.go

```diff
@@ -167,7 +169,7 @@ func (c *CallService) GetCallsInRange(start time.Time, end time.Time, data url.V
 	d.Del("Page") // just in case
 	if start != Epoch {
 		startFormat := start.UTC().Format(APISearchLayout)
-		d.Set("StartTime>", startFormat)
+		d.Set("StartTime>=", startFormat)
```

Comment:
Huh. The equals sign should be included when this turns into a string... the way you modified it you'd end up with 

```
StartTime>==2017-01-01
```

which is one too many equals signs...